app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Category;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+
+class CategoryController extends Controller
+{
+    public function index()
+    {
+        $categories = Category::orderby('id', 'desc')->paginate(15);
+
+        return view('Category.index', compact('categories'));
+    }
+
+    public function show()
+    {
+        $categories = Category::orderby('id')->get();
+
+        return view('Category.add', compact('categories'));
+    }
+
+    public function create(Request $request)
+    {
+        $category = new Category();
+        $category->name = $request->name;
+        $category->parent_id = $request->parent_id ?? null;
+        $category->slug = $request->slug;
+        $category->save();
+
+        return redirect('categories');
+    }
+
+    public function edit($id)
+    {
+        $category = Category::find($id);
+
+        return view('Category.edit', compact('category'));
```

Comment:
Tên view để tên thường nhé. 
```suggestion
        return view('category.edit', compact('category'));
```